man/systemd-analyze.xml

```diff
@@ -382,13 +389,44 @@ Normalized form: *-02-29 00:00:00
       </example>
     </refsect2>
 
+    <refsect2>
+      <title><command>systemd-analyze timestamp <replaceable>TIMESTAMP</replaceable>...</command></title>
+
+      <para>This command parses a timestamp (i.e. a single point in time) and outputs the normalized form and
+      the difference between this timestamp and now. The timestamp should adhere to the syntax documented in
+      <citerefentry><refentrytitle>systemd.time</refentrytitle><manvolnum>7</manvolnum></citerefentry>,
+      section "PARSING TIMESTAMPS".</para>
+
+      <example>
+        <title>Show parsing of timestamps</title>
+
+        <programlisting>$ systemd-analyze timestamp yesterday now tomorrow
+  Original form: yesterday
+Normalized form: Thu 2019-05-16 00:00:00 CEST
+       (in UTC): Wed 2019-05-15 22:00:00 UTC
```

Comment:
i think output in seconds since the epoch would make a ton of sense too:

```
UNIX seconds: @1558369282
```

In particular as we actually parse such `@` prefixed unix times ourselves...